### PR TITLE
fix error in developer.vue

### PR DESCRIPTION
**Pull request by tired6488**

### CODE REVIEW

**Comment by gayolGate** [comment]
thanks, it was my fault

File: NUXT/pages/mods/developer.vue [hunk 1]
```diff
@@ -106,10 +106,10 @@
       >
         <v-card-title class="text-h5">{{ lang.createentryfull }}</v-card-title>
         <v-card-text>
-          <v-text-field v-model="selectedKey" label="{{ lang.key }}" solo />
+          <v-text-field v-model="selectedKey" label="lang.key" solo />
           <v-text-field
             v-model="selectedKeyData"
-            label="{{ lang.value }}"
+            label="lang.value"
```
Comment (PickleNik) [inline NUXT/pages/mods/developer.vue:112]:
```diff
- label="lang.value"
+ :label="lang.value"

File: NUXT/pages/mods/developer.vue [hunk 2]
```diff
@@ -106,10 +106,10 @@
       >
         <v-card-title class="text-h5">{{ lang.createentryfull }}</v-card-title>
         <v-card-text>
-          <v-text-field v-model="selectedKey" label="{{ lang.key }}" solo />
+          <v-text-field v-model="selectedKey" label="lang.key" solo />
```
Comment (PickleNik) [inline NUXT/pages/mods/developer.vue:109]:
```diff
- label="lang.key"
+ :label="lang.key"
```

**Review by PickleNik — CHANGES_REQUESTED** [review]

**Comment by PickleNik** [comment]
https://vuejs.org/guide/essentials/template-syntax.html#attribute-bindings
You'll notice it highlights differently in the editor when you add a colon `:`

**Review by PickleNik — APPROVED** [review]

**Pull request merged**